lighthouse-core/report/handlebar-helpers.js

```diff
@@ -0,0 +1,182 @@
+/**
+ * @license
+ * Copyright 2017 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+/* global Intl */
+
+const Handlebars = require('handlebars');
+const marked = require('marked');
+
+const RATINGS = {
+  GOOD: {label: 'good', minScore: 75},
+  AVERAGE: {label: 'average', minScore: 45},
+  POOR: {label: 'poor'}
+};
+
+function calculateRating(value) {
+  let rating = RATINGS.POOR.label;
+  if (value >= RATINGS.GOOD.minScore) {
+    rating = RATINGS.GOOD.label;
+  } else if (value >= RATINGS.AVERAGE.minScore) {
+    rating = RATINGS.AVERAGE.label;
+  }
+  return rating;
+}
+
+const getTotalScore = aggregation => {
+  return Math.round(aggregation.total * 100);
+};
+
+const getItemRating = value => {
+  if (typeof value === 'boolean') {
+    return value ? RATINGS.GOOD.label : RATINGS.POOR.label;
+  }
+  return calculateRating(value);
+};
+
+const handlebarHelpers = {
+  // Converts a name to a link.
+  'nameToLink': name => {
```

Comment:
how does everyone feel about renaming the helper methods so they're all camelCase?

I don't feel strongly, but it would at least bring some consistency and avoid the quote thing (they aren't even consistent currently, obviously. some care camelCase, some are snake_case, and at least `is-bool` is kebab-case (which I just learned has a name))

The remaining question would be is e.g. `isNotEq` still readable in the html template, I guess.